src/hyperv/hyperv_api_v1.c

```diff
@@ -1594,20 +1594,28 @@ hyperv1DomainAttachSyntheticEthernetAdapter(virDomainPtr domain,
     char *connection__PATH = NULL;
     unsigned char guid[VIR_UUID_BUFLEN];
     virBuffer query = VIR_BUFFER_INITIALIZER;
-    Msvm_ComputerSystem_V1 *computer = NULL;
+    Msvm_VirtualSwitch_V1 *vSwitch = NULL;
     eprParam virtualSwitch_REF, ComputerSystem_REF;
     simpleParam virtualSwitch_Name, virtualSwitch_FriendlyName,
                 virtualSwitch_ScopeOfResidence;
     embeddedParam ResourceSettingData;
     invokeXmlParam *params = NULL;
     properties_t *NewResources = NULL;
 
+    /* get the vSwitch first so we can get data from it as needed */
```

Comment:
yeah that's left over from the previous iteration when I had the code work with network name instead of guid - this was looking up the vSwitch by name and then using guild down the line.